src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java

```diff
@@ -1448,8 +1482,20 @@ private void sendPostWithAccessCodeLetterToParty(CaseData caseData, String autho
                                                      Element<PartyDetails> party, String template,
                                                      String servedParty) {
 
+
+        sendPostWithAccessCodeLetterToParty(caseData,authorization,packDocs,bulkPrintDetails,party,template,servedParty,true);
```

Comment:
Can you please check with Sairam as it has been implmeneted and this refactoring is not needed.

Reply:
pushed the changes request to please kindly review it Thanks @shashisk97